pkg/action/executor/resize_container_util.go

```diff
@@ -66,8 +66,8 @@ func updateLimits(container *k8sapi.Container, patchCapacity k8sapi.ResourceList
 	}
 
 	if changed {
-		glog.V(2).Infof("Try to update container %v resource limit from %+v to %v",
-			container.Name, container.Resources.Limits, result)
+		glog.V(2).Infof("Try to update container %v for the workload controller [%v] resource limit from %+v to %v",
```

Comment:
nit: Change "Try to update container %v for the workload controller [%v] resource limit from %+v to %v" to "Try to update container %v limit from %+v to %v in workload controller [%v]"